nss-day-cohort-48/giffygram-hotbot
# Submit Post Button

#### Changes Made

1. Modified file `PostList.js` to include event listener that permanently saves a new post to the DOM / database.json. It works in conjunciton with a new createNewPost function in provider.js.
   ​

#### Steps to Review

1. Checkout this branch locally.
   ```
   git fetch --all
   git checkout gqg-submit-post-1
   ```
2. Open a new Terminal tab (⌘T) and navigate to the server directory.
3. Test app functionality.
   > Try adding a new post to the DOM!
4. View code file.
   > Confirm new post has been stringified and added to the database.
   > Confirm file modifications are present as indicated above.
   > Confirm no unused code or extraneous comments exist.
